diff --git a/docs/res b/docs/res
index d31dc929b..f82d1892e 160000
--- a/docs/res
+++ b/docs/res
@@ -1 +1 @@
-Subproject commit d31dc929bcd97e515ad529ea84d3e12268393bda
+Subproject commit f82d1892efe9fe2b53eedf04f0880844dc59fdd2
diff --git a/src-interface/recorder/recorder.cpp b/src-interface/recorder/recorder.cpp
index 91e0b434e..58924e070 100644
--- a/src-interface/recorder/recorder.cpp
+++ b/src-interface/recorder/recorder.cpp
@@ -600,10 +600,15 @@ namespace satdump
             ImGui::Spacing();
 
             if (!assume_recording)
+            {
+                ImGui::PushFont(style::bigFont);
                 ImGui::TextColored(style::theme.red, "IDLE");
+            }
             else
+            {
+                ImGui::PushFont(style::bigFont);
                 ImGui::TextColored(style::theme.green, "RECORDING");
-
+            }
             ImGui::Spacing();
 
             if (!assume_recording)
@@ -616,6 +621,7 @@ namespace satdump
                 if (ImGui::Button("Stop##stoprecording"))
                     stop_recording();
             }
+            ImGui::PopFont();
         }
 
         show_tracking = ImGui::CollapsingHeader("Tracking", tracking_started_cli ? ImGuiTreeNodeFlags_DefaultOpen : ImGuiTreeNodeFlags_None);
